services/clojars/clojars-base.js

```diff
@@ -7,9 +7,11 @@ const { BaseJsonService } = require('..')
 
 const clojarsSchema = Joi.object({
   downloads: nonNegativeInteger,
-  latest_version: Joi.string().required(),
-  latest_release: Joi.string().required(),
-}).required()
+  latest_version: Joi.string(),
+  latest_release: Joi.string(),
+})
+  .required()
+  .xor('latest_release', 'latest_version')
```

Comment:
I don't think we can/should use any exclusive clauses here, as it seems like a very common scenario for both of these fields to exist (this is why all the live tests are failing)

```json
{
  "jar_name": "prismic",
  "group_name": "prismic",
  "homepage": "http://prismic.io",
  "description": "Clojure kit for prismic.io API",
  "user": "erwan",
  "latest_version": "1.1",
  "latest_release": "1.1",
  "recent_versions": [...],
  "downloads": 126
}
```

Reply:
My assumption is that `latest_version` will always exist in the response for every valid jar (non-existent jars would return a 404 error) so it may just be a matter of removing that `xor` chain, keeping `latest_release` as optional, and making `latest_version` required

Reply:
There is also [`.or()`](https://github.com/hapijs/joi/blob/master/API.md#objectorpeers) which would make one or the other required, but both valid.